addons/delivery/views/delivery_price_rule_views.xml

```diff
@@ -18,12 +18,12 @@
                     </div>
                     <field name="currency_id" invisible="1"/>
                     <label for="list_base_price" string="Delivery Cost"/>
-                    <div>
-                        <field name="list_base_price" widget="monetary" class="oe_inline"/>
+                    <div class="row">
```

Comment:
I think it'd be simpler and more in line with what is generally done in form view to just handle the padding issue with a bootstrap class:
```diff
diff --git a/addons/delivery/views/delivery_price_rule_views.xml b/addons/delivery/views/delivery_price_rule_views.xml
index c45c618fd0d..fa9fb4df5c2 100644
--- a/addons/delivery/views/delivery_price_rule_views.xml
+++ b/addons/delivery/views/delivery_price_rule_views.xml
@@ -18,12 +18,12 @@
                     </div>
                     <field name="currency_id" invisible="1"/>
                     <label for="list_base_price" string="Delivery Cost"/>
-                    <div>
+                    <div class="d-flex">
                         <field name="list_base_price" widget="monetary" class="oe_inline"/>
                         +
-                        <field name="list_price" widget="monetary" class="oe_inline"/>
+                        <field name="list_price" widget="monetary" class="oe_inline ms-2"/>
                         *
-                        <field name="variable_factor" class="oe_inline"/>
+                        <field name="variable_factor" class="oe_inline ms-2"/>
                     </div>
                 </group>
             </form>
```